pkg/aks/create.go

```diff
@@ -109,8 +109,14 @@ func createManagedCluster(ctx context.Context, cred *Credentials, workplacesClie
 			VMSize:              containerservice.VMSizeTypes(np.VMSize),
 			Mode:                containerservice.AgentPoolMode(np.Mode),
 			OrchestratorVersion: np.OrchestratorVersion,
+			NodeLabels:          np.NodeLabels,
+			NodeTaints:          np.NodeTaints,
+		}
+		if np.MaxSurge != nil {
```

Comment:
can you add a unit test for these fields? Let's make sure we don't lose the functionality in future.

Reply:
I have added unit test